inst/htmlwidgets/r2d3.yaml

```diff
@@ -4,3 +4,8 @@ dependencies:
     src: htmlwidgets/lib
     script:
       - r2d3-render.js
+  - name: webcomponents-bundle
```

Comment:
Each dependency should typically be in it's own directory (as by default the entire directory is copied). So here I would create a "webcomponents" subdirectory within the `inst` directory (right where the d3 and d3-jetpack dependencies are) and use that as the source. The name of the dependency should also be "webcomponents" (rather than including the "-bundle" suffix) as that's the proper name of the JS library.

In terms of r2d3-render.js, I'm wondering if that code shouldn't just go into the existing r2d3.js source file (which is really just a stub right now).

Final thought: since subdirectories of `inst` show up as top-level directories in the installed package, we typically try not namespace them a bit. So I think the ideal directory structure for JS libraries is actually this:

```
inst/
  www/
     d3
     d3-jetpack
     webcomponents
```

Reply:
Opened a new PR to figure out this change.